src/map/atcommand.c

```diff
@@ -4454,6 +4454,38 @@ ACMD(unloadnpc)
 	return true;
 }
 
+/// Unload existing NPC within the NPC file and reload it.
+/// Usage: @reloadnpc npc/sample_npc.txt
+ACMD(reloadnpc)
+{
+	if (!*message) {
+		clif->message(fd, msg_fd(fd, 1385)); // Usage: @unloadnpcfile <file name>
+		return false;
+	} else if (npc->unloadfile(message) == true) {
+		clif->message(fd, msg_fd(fd, 1386)); // File unloaded. Be aware that mapflags and monsters spawned directly are not removed.
+
+		FILE *fp;
+		// check if script file exists
+		if ((fp = fopen(message, "r")) == NULL) {
```

Comment:
code ``fp = fopen(message, "r")`` can be moved into line with ``FILE *fp`` as ``FILE *fp = fopen(message, "r")``

Reply:
done